test/verify/check-system-services

```diff
@@ -329,6 +329,15 @@ Unit=test.service
 
         b.go("#/")
 
+        # Check that listen property is shown for socket units
+        self.pick_tab(3)
+        b.click(self.svc_sel("dbus.socket"))
+        b.wait_visible("#listen")
```

Comment:
`wait_in_text("#listen", "/run/dbus/")` ? to make sure it doesn't show something empty or bogus?

Reply:
I have tihs self.assertIn("Stream", b.text("#listen .pf-c-description-list__text")), what you commented on seems outdated version.

Reply:
I can add the whole thing, /run/dbus (Stream) though, I was just trying to avoid distribution specific differences.

Reply:
I reviewed the commits one by one indeed; `/run/` isn't distribution specific, that should be perfectly safe. "dbus" certainly ought to appear somewhere as well.